apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -157,13 +165,36 @@ export const EventSetupTab = (
       return true;
     });
 
+    const defaultValue = isManagedEventType
+      ? locationOptions.find((op) => op.label === t("default"))?.options[0]
```

Comment:
We need to show the inserted value at 0 (referencing user's default location) for a managed event